src/maintenance/azext_maintenance/commands.py

```diff
@@ -0,0 +1,56 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import CliCommandType
+
+from ._client_factory import (cf_maintenance_configurations, cf_maintenance_updates, cf_configuration_assignments, cf_apply_updates)
+
+
+def load_command_table(self, _):
+
+    maintenance_configurations_mgmt_util = CliCommandType(
+        operations_tmpl='azext_maintenance.maintenance.operations.maintenance_configurations_operations#MaintenanceConfigurationsOperations.{}',
+        client_factory=cf_maintenance_configurations,
+        client_arg_name='self'
+    )
+
+    maintenance_updates_mgmt_util = CliCommandType(
+        operations_tmpl='azext_maintenance.maintenance.operations.updates_operations#UpdatesOperations.{}',
+        client_factory=cf_maintenance_updates,
+        client_arg_name='self'
+    )
+
+    configuration_assignments_mgmt_util = CliCommandType(
+        operations_tmpl='azext_maintenance.maintenance.operations.configuration_assignments_operations#ConfigurationAssignmentsOperations.{}',
+        client_factory=cf_configuration_assignments,
+        client_arg_name='self'
```

Comment:
It seems that client_arg_name='self' is unnecessary.

Reply:
Yes, I've removed it now